src/components/playlist-button/playlist-button.js

```diff
@@ -46,44 +53,57 @@ class PlaylistButton extends Component {
    */
   render(props: any): React$Element<any> | void {
     const item = props.playlist[props.type];
+    const showPreview = item && item.sources && (item.sources.poster || (item.sources.metadata && item.sources.metadata.name));
     return (
       <div className={[style.controlButtonContainer, style.controlPlaylistButton].join(' ')}>
-        {item && item.sources && (item.sources.poster || (item.sources.metadata && item.sources.metadata.name)) ? (
+        {showPreview ? (
           <div className={style.posterPreview}>
             <div className={style.posterPreviewText}>
-              <Localizer>
-                <div className={style.posterPreviewTextTitle}>
-                  <Text id={props.type === 'prev' ? 'playlist.prev' : 'playlist.up_next'} />
-                </div>
-              </Localizer>
+              <div className={style.posterPreviewTextTitle}>
+                {props.type === 'prev' ? this.props.playlistPrevText : this.props.playlistUpNextText}
+              </div>
               <div className={style.posterPreviewTextName}>{`${item.sources.metadata ? item.sources.metadata.name : ''}`}</div>
             </div>
             <div className={style.posterPreviewImg} style={`background-image: url(${item.sources.poster});`} />
           </div>
         ) : (
           undefined
         )}
-        <Localizer>
-          <button
-            disabled={!item}
-            tabIndex="0"
-            aria-label={<Text id={`controls.${props.type}`} />}
-            className={`${style.controlButton}`}
-            onClick={() => this.onClick()}>
-            {props.type === 'prev' ? (
-              <div>
-                <Icon type={IconType.Prev} />
-              </div>
-            ) : (
-              <div>
-                <Icon type={IconType.Next} />
-              </div>
-            )}
-          </button>
-        </Localizer>
+        {showPreview ? (
```

Comment:
don't you need tooltip here as well?

Reply:
No. when there is a preview it is ugly and unnecessary to have also a tooltip